recipe/meta.yaml

```diff
@@ -1,88 +1,79 @@
 # Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
 # It has been subsequently adapted for automated building with conda-forge.
 
-{% set version = "2.2.2" %}
+{% set version = "3.0.0" %}
 
 package:
   name: matplotlib
   version: {{ version }}
 
 source:
   url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
-  sha256: 3a8236e6747dbb48c342ee194a8ced2805205cbd718899b284e4d617189cd397
+  sha256: 4ab565581e3012fec06f93fc181d719a6f214f34c5b2a9f808741f0097507ea4
   patches:
     # Find libpng on Windows.
     - setupext.patch  # [win]
-    # A patch to make Qt the default backend.
-    - cfg_qt5agg.patch  # [not osx]
-    # Removes framework check from osx backend (seems unneeded)
-    - osx-frame.patch  # [osx]
 
 build:
   number: 200
+  skip: true  # [py<35]
 
 requirements:
```

Comment:
Maybe you need a python dep here (and not just under the output) for the CI jobs to run separately? Not sure.